### PR TITLE
Allow to specify simulator OS version from cli

### PR DESCRIPTION
**Pull request by tomkur**
When multiple simulator runtimes are available, this change will make it possible to specify which one to run. So assuming you have e.g. iOS 9.0 and 9.3 runtimes, all following calls will now work:
```
react-native run-ios --simulator "iPhone 6s (9.0)"
react-native run-ios --simulator "iPhone 6s (9.3)"
react-native run-ios --simulator "iPhone 6s"
```

### CODE REVIEW

**Comment by facebook-github-bot-1** [comment]
By analyzing the blame information on this pull request, we identified @mroswald and @frantic to be potential reviewers.

**Comment by facebook-github-bot-1** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by mroswald** [comment]
LGTM for me, nice feature!

**Comment by facebook-github-bot-5** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: local-cli/runIOS/runIOS.js [hunk 1]
```diff
@@ -121,10 +121,14 @@ function _runIOS(argv, config, resolve, reject) {
...
 function matchingSimulator(simulators, simulatorName) {
   for (let i = simulators.length - 1; i >= 0; i--) {
-    if (simulators[i].name === simulatorName) {
+    if (simulators[i].name === simulatorName || calcSimulatorFullName(simulators[i]) === simulatorName) {
       return simulators[i];
     }
   }
 }
 
+function calcSimulatorFullName(simulator) {
```
Comment (javache) [inline local-cli/runIOS/runIOS.js]: maybe rename this to `formattedSimulatorName`?

**Comment by frantic** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-7** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/541203162754480/int_phab) to review.

**Pull request closed without merging**